swirl_lm/base/driver.py

```diff
@@ -56,6 +56,11 @@
     'The number of cycles between which full variables may be dumped to file.',
     allow_override=True,
 )
+NEVER_LOG_FULL_STATE = flags.DEFINE_bool(
+  'never_log_full_state',
```

Comment:
Perhaps: 'never_save_full_state' or 'never_dump_full_state' ? 
We reserve "log" for the actual logging. Thanks,

Reply:
Another thing is: perhaps let's change it to "skip_log_full_state" -- to avoid double-negative?

Reply:
Yes that is a good idea